src/pages/steps/ScopeStep.tsx

```diff
@@ -147,15 +158,12 @@ const ScopeStep = <TFormData extends FormDataUnion>({
 };
 
 const scopeStepConfiguration: StepsConfiguration<FormDataUnion> = {
-  Component: ScopeStep,
+  // eslint-disable-next-line react/display-name
+  Component: (props) => <ScopeStep {...props} />,
```

Comment:
I think this can just remain
```suggestion
Component: ScopeStep,
```

Reply:
True, fixed in c2c9075